src/main/resources/templates/teacher/teacher_list.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4">
+<html lang="en">
+<head>
+    <title>Teacher page</title>
+    <meta charset="utf-8">
+    <!--To ensure proper rendering and touch zooming-->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--Font Awesome-->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+
+    <!--Latest minified Bootstrap, JQuery, Moment-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+    <script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/jquery/latest/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
+    <!-- Custom styles for this template -->
+    <link rel="stylesheet" type="text/css" href="../css/main.css"/>
+</head>
+
+<body class="text-center">
+
+<!--Header content here-->
+<header class="header">
+    <div th:replace="fragments/header :: header(activeTab='teacher')"/>
+</header>
+
+<!-- Begin page content -->
+<main role="main" class="container">
+    <a th:href="@{/teacher/add}" class="float-right btn btn-outline-success">Add New</a>
+    <h4 class="h3 mb-3 font-weight-normal" th:inline="text"><small>Teachers</small></h4>
+    <div class="table-responsive" th:if="${not #lists.isEmpty(teachers)}">
+        <table class="col-lg-9 table table-hover">
+            <thead>
+            <tr>
+                <th>Lastname</th>
+                <th>Firstname</th>
+                <th>Patronym</th>
+                <th>Officer</th>
+                <th>Phone</th>
+                <th>Edit</th>
+                <th>Delete</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr th:each="teacher : ${teachers}">
+                <td th:text="${teacher.getLastName()}">Surname</td>
```

Comment:
You can use *{getLastName} after ${...}

Reply:
*{lastName} when you point a field or ${teacher.getLastName()} to call a method or ${teacher.lastName} for a variable - it's ok to use here any kind of Thymeleaf standard expressions.